che-theia-plugins.yaml

```diff
@@ -768,3 +768,7 @@ plugins:
     repository:
       url: 'https://github.com/BroadcomMFD/cobol-control-flow'
     extension: https://open-vsx.org/api/BroadcomMFD/ccf/0.4.0/file/BroadcomMFD.ccf-0.4.0.vsix
+  - repository:
+      url: 'https://github.com/sgwozdz/jenkinsfile-support'
+      revision: '1.2.0'
+    extension: 'https://disaster37.github.io/vscode-plugins/recode4dev.jenkinsfile-1.2.0.vsix'
```

Comment:
Please add a new line

Reply:
Hi, i doesn't understand what I need to do. What to you mean by `Please add new line `?

Reply:
It is a good practice to add an empty line at the end of the files.
For following review, it will be easier to compare.
(it also allows easier manipulating of files when concatenating them)

Github is suggesting to add this new line by providing a red circle with a minus inside marker.

Reply:
You can see it when open Files changed 
![screenshot-github com-2021 07 16-12_10_19](https://user-images.githubusercontent.com/1271546/125923852-02e2347d-df5b-4e86-985d-9c97421eb9b9.png)

To avoid that you just need to add empty line into the end of the file. 

Also we have PR check which validates newline and whitespace rules in files:
![screenshot-github com-2021 07 16-12_14_10](https://user-images.githubusercontent.com/1271546/125924243-8249f802-e1f5-4091-bc7e-4b09dc7a264a.png)